apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json

```diff
@@ -58,6 +58,14 @@
                                                 ],
                                                 "title": "type",
                                                 "type": "string"
+                                            },
+                                            "unitType": {
+                                                "enum": [
+                                                    "flat",
+                                                    "parking"
```

Comment:
What about `apartment`? It is a different kind of unit formally.

Reply:
@AntonAL this types will be expanded later, for now we need only two types